src/pages/workspaces/workspace/Workflows.js

```diff
@@ -237,9 +289,17 @@ const FindWorkflowModal = ajaxCaller(class FindWorkflowModal extends Component {
     const { namespace, name, snapshotId } = selectedWorkflow
 
     this.setState({ selectedWorkflow })
+
     try {
       const selectedWorkflowDetails = await Methods.method(namespace, name, snapshotId).get()
-      this.setState({ selectedWorkflowDetails })
+
+      const methodAjax = Ajax().Methods.method(selectedWorkflow.namespace, selectedWorkflow.name, selectedWorkflow.snapshotId)
+
+      const config = _.maxBy('snapshotId', await methodAjax.configs())
+
+      this.setState({ selectedWorkflowDetails, methodAjax })
```

Comment:
`methodAjax` is a temporary object from the ajax code, and shouldn't be stored in state.